devtools-ci-index.yaml

```diff
@@ -649,7 +649,7 @@
             exit $rtn_code
     publishers:
         - archive:
-            artifacts: ./perfcake-7.4/perfcake-chart
+            artifacts: ./devtools-performance-results
```

Comment:
@pmacik Is `perfcake-chart` a directory ? and what about `devtools-performance-results` ?